components/automate-chef-io/layouts/shortcodes/table.html

```diff
@@ -0,0 +1,3 @@
+<div class="table" style="overflow-x:auto;">
+  {{ .Inner }}
+</div>
```

Comment:
@kagarmoe I left the style here so I can chat with you about where this would live.

Reply:
Maybe: 

```
<div class="table"> 
  {{- .Inner | markdownify -}}
</div>
```

And then somewhere in the scss:
```
.table {
    overflow-x:auto; 
}
```
Once you put markdown text inside a div tag (or any other html tag I believe) blackfriday treats it like html so you have to explicitly tell it to render the text as markdown. Hence the `markdownify` command. With the css, I think it's better to have it one location instead of added to the html templates. My 2 cents.

Also, I'd add the shortcode (however it's named) to the Hugo theme so it can be used on other Hugo sites if it were ever needed.

Reply:
Yes. That is the question. I have no idea where it would go in the hugo theme @IanMadd - I wanted to defer to you two on naming and file organization since you deal with the docs. There isn't css in the themes main files, everything is imported so I'm just making sure I need to create a directory and a file or two to import into that file to keep that convention and make sure I'm not missing anything to y'alls knowledge. 

Also super Interesting it knows it's markdown without that "markdownify" tag added I had read that in the black friday docs as well, but examples on this work around (for this common hugo table problem) all had it without. Wasn't sure if it just works because it's a table?

Reply:
That's odd. I just tried your PR without `markdownify` and the tables look like raw text. 

<img width="938" alt="Screen Shot 2019-10-15 at 10 20 54 AM" src="https://user-images.githubusercontent.com/8138169/66841965-74e6ca80-ef38-11e9-9870-e0f20c9ac203.png">

Ok, it took a bit of sleuthing but I think I figured it out. I think we're running [different versions](https://discourse.gohugo.io/t/0-55-what-am-i-doing-wrong/18034) of Hugo. Check which version you have. There was an update to this behavior starting with 0.55. Before that you didn't have to add `markdownify`, afterwards you do. I added ```{{ $_hugo_config := `{ "version": 1 }` }}``` to your shortcode and it gave the behavior that you describe.

Yeah, I'd go with @kagarmoe's suggestion and call it `responsive-table`. If you want I could open a PR against the Chef Hugo theme and add a "responsive-table" shortcode and the css to sass/automate/_homepage.scss.

Reply:
Perfect!

Reply:
Super catch on the versions thing @IanMadd ! I will put up a PR on the hugo side. Thanks for pointing me in the right direction.